package.json

```diff
@@ -6,81 +6,77 @@
   "engines": {
     "node": ">=18 <21"
   },
-  "main": "index.js",
-  "prettier": "eslint-config-vinta/prettier",
+  "browserslist": "> 0.25%, not dead",
```

Comment:
I guess we didn't use that before and were generating larger bundles unnecessarily.